software/e-label.yml

```
@@ -0,0 +1,17 @@
+name: "E-Label"
+website_url: "https://filipecarneiro.github.io/ELabel/"
+source_code_url: "https://github.com/filipecarneiro/ELabel"
+description: "Open-source solution for electronic labels, with QR Codes, on wine bottles sold within the European Union. This initiative seeks to enhance transparency, efficiency, and information accessibility in the wine industry."
+licenses:
+  - MIT
+platforms:
+  - .NET
+  - C#
+  - Docker
+  - K8S
```

The deployment sections only mention docker as a way to install this software.
```suggestion
  - Docker
```